modular_skyrat/modules/morenarcotics/code/items/bricks/cocaine.dm

```diff
@@ -0,0 +1,61 @@
+/obj/item/reagent_brick/cocaine
+	name = "cocaine brick"
+	desc = "A brick of cocaine. Good for transport!"
+	icon = 'modular_skyrat/modules/morenarcotics/icons/crack.dmi'
+	icon_state = "cocainebrick"
+	packed_item = /obj/item/snortable/cocaine
+	brick_reagent = /datum/reagent/drug/cocaine/powder_cocaine
+	reagent_amount = 25
+
+/datum/crafting_recipe/cocaine_brick
+	result = /obj/item/reagent_brick/cocaine
+	reqs = list(/obj/item/snortable/cocaine = 5)
+	parts = list(/obj/item/snortable/cocaine = 5)
+	time = 20
+	category = CAT_CHEMISTRY //i might just make a crafting category for drugs at some point
+
+
+/obj/item/reagent_brick/crack
+	name = "crack brick"
+	desc = "A brick of crack. Good for transport!"
+	icon = 'modular_skyrat/modules/morenarcotics/icons/crack.dmi'
+	icon_state = "crackbrick"
+	packed_item = /obj/item/smokable/crack
+	packed_amount = 4
+	brick_reagent = /datum/reagent/drug/cocaine/freebase_cocaine
+	reagent_amount = 40
+
+/datum/crafting_recipe/crack_brick
+	result = /obj/item/reagent_brick/crack
+	reqs = list(/obj/item/smokable/crack = 4)
+	parts = list(/obj/item/smokable/crack = 4)
+	time = 20
+	category = CAT_CHEMISTRY //i might just make a crafting category for drugs at some point
+	var/list/purity_hash = list() // im sorry man but there is literally no other fucking idea i have for how to pull this off
```

Comment:
This is going to need some documentation, if you're going to keep it.
But if you're not sure how to do this better, why not ask about it in #development-code?